DoubleActionCylinder/src/main/java/org/usfirst/frc2130/DoubleActionCylinder/subsystems/PneumaticSubsystem.java

```diff
@@ -66,5 +79,32 @@ public void periodic() {
     // Put methods for controlling this subsystem
     // here. Call these from Commands.
 
+
+    //There are three functions here. Neutral puts the solenoid in the "off" state, which makes it neutral, so it can be moved
+    //Out makes the solenoid go forward, and in makes the solenoid go backwards
+    public void solenoidNeutral(){
+        solenoidOne.set(Value.kOff);
+        
+    }
```

Comment:
You never use this command, you want to either call it or remove it. (By remove it you can either actually remove it or you can just comment it out)